lib/openscap_parser/test_result.rb

```diff
@@ -4,10 +4,39 @@ module OpenscapParser
   module TestResult
     def self.included(base)
       base.class_eval do
-        private
+        def score
+          @score ||= test_result_node &&
+            test_result_node.search('score').text.to_f
+        end
+
+        def start_time
+          @start_time ||= test_result_node &&
+            DateTime.parse(test_result_node['start-time'])
+        end
+
+        def end_time
+          @end_time ||= test_result_node &&
+            DateTime.parse(test_result_node['end-time'])
+        end
+
+        def test_result_profiles
+          @test_result_profiles ||= test_result_profile_nodes.inject({}) do |profiles, profile_node|
+            profiles[profile_node['id']] = profile_node.at_css('title').text
+
+            profiles
+          end
+        end
 
         def test_result_node
-          @test_result_node ||= @parsed_xml.at_css('TestResult')
+          @test_result_node ||= xpath_node('.//TestResult')
+        end
+
+        private
```

Comment:
I'd be willing to drop this `private` if you think `test_result_profile_nodes` could be useful to users of this gem.

Reply:
I think you can keep it as is for now.